twentytwenty-blocks/twentytwenty-styles/editor-style-block.css

```diff
@@ -163,21 +168,21 @@ Inter variable font. Usage:
 /* CUSTOM COLORS */
 
 :root .has-accent-color {
-	color: #cd2653;
+	color: var(--wp--preset--color--primary);
 }
 
 :root .has-accent-background-color {
-	background-color: #cd2653;
+	background-color: var(--wp--preset--color--primary);
 	color: #fff;
 }
 
 :root .has-primary-color {
-	color: #000;
+	color: var(--wp--preset--color--primary);
 }
 
 :root .has-primary-background-color {
-	background-color: #000;
-	color: #f5efe0;
+	background-color: var(--wp--preset--color--text);
+	color: var(--wp--preset--color--primary);
```

Comment:
There is some strangeness when you apply a background color and text color to buttons with the "Fill" style:

**Editor**
<img width="860" alt="Screen Shot 2020-07-21 at 9 56 20 AM" src="https://user-images.githubusercontent.com/5375500/88063895-94023080-cb38-11ea-8dd8-ba4019950cee.png">

**Front-end**
<img width="235" alt="Screen Shot 2020-07-21 at 9 56 26 AM" src="https://user-images.githubusercontent.com/5375500/88063905-982e4e00-cb38-11ea-8cfa-54412e0a8708.png">

Reply:
I had to do a few weird things to get this working: 

1. Rename the "Text" color to "Foreground". Otherwise, the class to indicate that this color was being used was `has-text-color`. This also happens to be the generic classname that's added to blocks when they have _any_ text color assigned. That caused some conflicting styles.
2. I had to rewrite some of the styles, because they seemed to be wrong. 
3. I had to add some extra specificity to get the custom color rules working for buttons. 

It's possible I chopped out a little more code than was necessary here, but it generally seemed to work for me.